### PR TITLE
sql: update br docs about some sql options

### PR DESCRIPTION
**Pull request by RidRisR**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/docs/pull/17416

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-backup.md [hunk 1]
```diff
@@ -125,9 +126,9 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
 
 如果你需要减少网络带宽占用，可以通过 `RATE_LIMIT` 来限制每个 TiKV 节点的平均上传速度。
 
-默认情况下，每个 TiKV 节点上运行 4 个备份线程。可以通过 `CONCURRENCY` 选项来调整这个值。
+在备份完成之前，`BACKUP` 将对集群上的数据执行校验和以验证数据正确性。如果您确信此检查不必要，可以使用 `CHECKSUM` 来禁用该检查。使用 `CHECKSUM_CONCURRENCY` 来控制单个表中校验和的并发性（默认值为4）。
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:129]:
```suggestion
在备份完成之前，`BACKUP` 将对集群上的数据执行校验和，以验证数据正确性。如果你确认此检查不必要，可以使用 `CHECKSUM` 禁用该检查。如果需要控制单个表中校验和的并发性，可使用 `CHECKSUM_CONCURRENCY`，默认值为 4。
```

File: sql-statements/sql-statement-backup.md [hunk 2]
```diff
@@ -125,9 +126,9 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
...
 
-在备份完成之前，`BACKUP` 将对集群上的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以通过 `CHECKSUM` 选项禁用这一步骤。
+使用 `CONCURRENCY` 来指定 BR 可以同时执行的备份表和索引的任务数量。该参数控制 BR 的线程池大小，优化备份操作的性能和效率。根据备份类型不同，一个任务代表一个表范围或一个索引范围。如果有一个表带有一个索引，则会有两个任务来备份这个表。如果您需要备份许多表或索引，应增加此值。（默认值为4）
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:131]:
```suggestion
要指定 BR 可以同时执行的备份表和索引的任务数量，可使用 `CONCURRENCY`。该参数控制 BR 的线程池大小，可以优化备份操作的性能和效率。根据备份类型不同，一个任务代表一个表范围或一个索引范围。如果有一个表带有一个索引，则会有两个任务来备份这个表。如果你需要备份许多表或索引，应调大该参数的值，默认值为 4。
```

File: sql-statements/sql-statement-restore.md [hunk 3]
```diff
@@ -42,6 +42,7 @@ RestoreOption ::=
     "RATE_LIMIT" '='? LengthNum "MB" '/' "SECOND"
 |   "CONCURRENCY" '='? LengthNum
 |   "CHECKSUM" '='? Boolean
+|   "CHECKSUM_CONCURRENCY '='? LengthNum
```
Comment (lilin90) [inline sql-statements/sql-statement-restore.md:45]:
```suggestion
|   "CHECKSUM_CONCURRENCY" '='? LengthNum
```

File: sql-statements/sql-statement-restore.md [hunk 4]
```diff
@@ -120,9 +121,7 @@ RESTORE DATABASE * FROM 's3://example-bucket-2020/backup-05/'
 
 如果你需要减少网络带宽占用，可以通过 `RATE_LIMIT` 来限制每个 TiKV 节点的平均下载速度。
 
-默认情况下，每个 TiKV 节点上运行 128 个恢复线程。可以通过 `CONCURRENCY` 选项来调整这个值。
-
-在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以通过 `CHECKSUM` 选项禁用这一步骤。
+在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果您确信此检查不必要，可以使用 `CHECKSUM` 来禁用该检查。使用 `CHECKSUM_CONCURRENCY` 来控制单个表中校验和的并发性（默认值为4）。
```
Comment (lilin90) [inline sql-statements/sql-statement-restore.md:124]:
```suggestion
在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果你确认此检查不必要，可以使用 `CHECKSUM` 禁用该检查。如果需要控制单个表中校验和的并发性，可使用 `CHECKSUM_CONCURRENCY`，默认值为 4。
```

**Comment by lilin90** [comment]
@RidRisR Please take a look at the comments and invite a tech reviewer.

**Review by YuJuncen — APPROVED** [review]

File: sql-statements/sql-statement-backup.md [hunk 5]
```diff
@@ -125,9 +125,9 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
 
 如果你需要减少网络带宽占用，可以通过 `RATE_LIMIT` 来限制每个 TiKV 节点的平均上传速度。
 
-默认情况下，每个 TiKV 节点上运行 4 个备份线程。可以通过 `CONCURRENCY` 选项来调整这个值。
+在备份完成之前，`BACKUP` 将对集群上的数据执行校验和以验证数据正确性。如果您确信此检查不必要，可以使用 `CHECKSUM` 来禁用该检查。
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:128]:
```suggestion
在备份完成之前，`BACKUP` 将对集群上的数据执行校验和以验证数据正确性。如果你确信无需进行校验，可以使用 `CHECKSUM` 参数禁用该检查。
```

File: sql-statements/sql-statement-restore.md [hunk 6]
```diff
@@ -120,9 +120,7 @@ RESTORE DATABASE * FROM 's3://example-bucket-2020/backup-05/'
 
 如果你需要减少网络带宽占用，可以通过 `RATE_LIMIT` 来限制每个 TiKV 节点的平均下载速度。
 
-默认情况下，每个 TiKV 节点上运行 128 个恢复线程。可以通过 `CONCURRENCY` 选项来调整这个值。
-
-在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以通过 `CHECKSUM` 选项禁用这一步骤。
+在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果您确信此检查不必要，可以使用 `CHECKSUM` 来禁用该检查。
```
Comment (lilin90) [inline sql-statements/sql-statement-restore.md:123]:
```suggestion
在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以使用 `CHECKSUM` 参数禁用该检查。
```

File: sql-statements/sql-statement-backup.md [hunk 7]
```diff
@@ -125,9 +125,9 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
 
 如果你需要减少网络带宽占用，可以通过 `RATE_LIMIT` 来限制每个 TiKV 节点的平均上传速度。
 
-默认情况下，每个 TiKV 节点上运行 4 个备份线程。可以通过 `CONCURRENCY` 选项来调整这个值。
+在备份完成之前，`BACKUP` 将对集群上的数据执行校验和以验证数据正确性。如果你确信无需进行校验，可以使用 `CHECKSUM` 参数禁用该检查。
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:128]:
```suggestion
在备份完成之前，`BACKUP` 将对集群上的数据执行校验和以验证数据正确性。如果你确信无需进行校验，可以通过将 `CHECKSUM` 参数设置为 `FALSE` 来禁用该检查。
```

File: sql-statements/sql-statement-backup.md [hunk 8]
```diff
@@ -125,9 +125,9 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
...
 
-在备份完成之前，`BACKUP` 将对集群上的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以通过 `CHECKSUM` 选项禁用这一步骤。
+要指定 BR 可以同时执行的备份表和索引的任务数量，可使用 `CONCURRENCY`。该参数控制 BR 的线程池大小，可以优化备份操作的性能和效率。根据备份类型不同，一个任务代表一个表范围或一个索引范围。如果有一个表带有一个索引，则会有两个任务来备份这个表。参数 `CONCURRENCY` 的默认值为 4，如果你要备份许多表或索引，需调大该参数的值。
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:130]:
```suggestion
要指定 BR 可以同时执行的备份表和索引的任务数量，可使用 `CONCURRENCY`。该参数控制 BR 的线程池大小，可以优化备份操作的性能和效率。根据备份类型不同，一个任务代表一个表范围或一个索引范围。如果有一个表带有一个索引，则会有两个任务来备份这个表。参数 `CONCURRENCY` 的默认值为 `4`，如果你要备份许多表或索引，需调大该参数的值。
```

File: sql-statements/sql-statement-restore.md [hunk 9]
```diff
@@ -120,9 +120,7 @@ RESTORE DATABASE * FROM 's3://example-bucket-2020/backup-05/'
 
 如果你需要减少网络带宽占用，可以通过 `RATE_LIMIT` 来限制每个 TiKV 节点的平均下载速度。
 
-默认情况下，每个 TiKV 节点上运行 128 个恢复线程。可以通过 `CONCURRENCY` 选项来调整这个值。
-
-在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以通过 `CHECKSUM` 选项禁用这一步骤。
+在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以使用 `CHECKSUM` 参数禁用该检查。
```
Comment (lilin90) [inline sql-statements/sql-statement-restore.md:123]:
```suggestion
在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以通过将 `CHECKSUM` 参数设置为 `FALSE` 来禁用该检查。
```

File: sql-statements/sql-statement-backup.md [hunk 10]
```diff
@@ -125,9 +125,9 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
 
 如果你需要减少网络带宽占用，可以通过 `RATE_LIMIT` 来限制每个 TiKV 节点的平均上传速度。
 
-默认情况下，每个 TiKV 节点上运行 4 个备份线程。可以通过 `CONCURRENCY` 选项来调整这个值。
+在备份完成之前，`BACKUP` 将对集群上的数据执行校验和以验证数据正确性。如果你确信无需进行校验，可以通过将 `CHECKSUM` 参数设置为 `FALSE` 来禁用该检查。
```
Comment (hfxsd) [inline sql-statements/sql-statement-backup.md:128]:
```suggestion
在备份完成之前，`BACKUP` 将对集群上的数据执行校验和以验证数据正确性。如果你确定无需进行校验，可以通过将 `CHECKSUM` 参数设置为 `FALSE` 来禁用该检查。
```

File: sql-statements/sql-statement-restore.md [hunk 11]
```diff
@@ -120,9 +120,7 @@ RESTORE DATABASE * FROM 's3://example-bucket-2020/backup-05/'
 
 如果你需要减少网络带宽占用，可以通过 `RATE_LIMIT` 来限制每个 TiKV 节点的平均下载速度。
 
-默认情况下，每个 TiKV 节点上运行 128 个恢复线程。可以通过 `CONCURRENCY` 选项来调整这个值。
-
-在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以通过 `CHECKSUM` 选项禁用这一步骤。
+在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以通过将 `CHECKSUM` 参数设置为 `FALSE` 来禁用该检查。
```
Comment (hfxsd) [inline sql-statements/sql-statement-restore.md:123]:
```suggestion
在恢复完成之前，`RESTORE` 将对备份文件中的数据进行校验，以验证数据的正确性。如果你确定无需进行校验，可以通过将 `CHECKSUM` 参数设置为 `FALSE` 来禁用该检查。
```

**Review by hfxsd — APPROVED** [review]

File: sql-statements/sql-statement-backup.md [hunk 12]
```diff
@@ -125,9 +125,9 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
...
 
-在备份完成之前，`BACKUP` 将对集群上的数据进行校验，以验证数据的正确性。如果你确信无需进行校验，可以通过 `CHECKSUM` 选项禁用这一步骤。
+要指定 BR 可以同时执行的备份表和索引的任务数量，可使用 `CONCURRENCY`。该参数控制 BR 的线程池大小，可以优化备份操作的性能和效率。根据备份类型不同，一个任务代表一个表范围或一个索引范围。如果有一个表带有一个索引，则会有两个任务来备份这个表。参数 `CONCURRENCY` 的默认值为 `4`，如果你要备份许多表或索引，需调大该参数的值。
```
Comment (hfxsd) [inline sql-statements/sql-statement-backup.md:130]: 需要给个 `CONCURRENCY` 的取值范围吗？

Comment (lilin90) [reply sql-statements/sql-statement-backup.md:130]: @RidRisR PTAL

Comment (RidRisR) [reply sql-statements/sql-statement-backup.md:130]: 不需要 这个取值范围取决于硬件性能

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**